### PR TITLE
0.9.0 generate new docs

### PR DESCRIPTION
**Pull request by carolineechen**

### CODE REVIEW

File: 0.9.0/.buildinfo [hunk 1]
```diff
@@ -1,4 +0,0 @@
-# Sphinx build info version 1
```
Comment (carolineechen) [inline 0.9.0/.buildinfo:1]: should this file have been deleted (as is the case from following README) or just updated?

File: 0.9.0/_static/pygments.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 pre { line-height: 125%; }
-td.linenos pre { color: #000000; background-color: #f0f0f0; padding-left: 5px; padding-right: 5px; }
```
Comment (carolineechen) [inline 0.9.0/_static/pygments.css:2]: same with this file -- is this okay or should it be changed back

Comment (mthrok) [reply 0.9.0/_static/pygments.css:2]: Can you enable your gh-pages build and skim through the doc to see if there is any obvious layout change? If not we can keep the change.

File: 0.9.0/.buildinfo (hunk 1 above)
Comment (mthrok) [reply 0.9.0/.buildinfo:1]: I think it's okay either way, as long as the resulting page is rendered correctly. We can come back to this if it affects GitHub's page deployment.

**Comment by mthrok** [comment]
Did you install the dependencies as directed in doc build process? i.e. `pip install -r requirements.txt` in doc directory. If you did that, then things should be fine. The differences on CSS and such might be coming from the upstream repo.

**Review by mthrok — APPROVED** [review]
Token approval

**Pull request merged**